lms/templates/main.html

```diff
@@ -137,7 +137,7 @@
     % endif
 
     <div class="content-wrapper" id="content">
-      ${self.body()}
+      ${next.body()}
```

Comment:
This is the one thing I'm not sure about, due to lack of knowledge on templating